api/protocol/protocol.go

```diff
@@ -28,6 +28,11 @@ const (
 	logLevel           = "protocol=INFO"
 	WSConnectTimeout   = 20 * time.Second
 	WSHandshakeTimeout = 15 * time.Second
+
+	ErrCodeUnknown    = 0
+	ErrCodeNotFound   = 1
+	ErrCodeBadRequest = 2
+	ErrCodeInternal   = 3
```

Comment:
I think moving these to a separate `const` block would improve readability.
Also, I think `iota` could be used here.